Makefile-test.mk

```diff
@@ -59,6 +59,8 @@ IMAGE_TAG ?= $(IMAGE_REPO):$(GIT_TAG)
 # JobSet Version
 JOBSET_VERSION = $(shell $(GO_CMD) list -m -f "{{.Version}}" sigs.k8s.io/jobset)
 KUBEFLOW_VERSION = $(shell $(GO_CMD) list -m -f "{{.Version}}" github.com/kubeflow/training-operator)
+# Mismatch between manifest semver tag and docker release tag, latter has no leading 'v'
+KUBEFLOW_MPI_VERSION = $(shell $(GO_CMD) list -m -f "{{.Version}}" github.com/kubeflow/mpi-operator | sed 's/^v//')
```

Comment:
It's very easy to remove `v` from the string on bash. Can we move this logic to place where we really need it? 

```bash
export KUBEFLOW_MPI_IMAGE=mpioperator/mpi-operator:${KUBEFLOW_MPI_VERSION/v}
```

Reply:
Sure, we could